resources/icarMilkPredictionResource.json

```
@@ -0,0 +1,27 @@
+{
+  "description": "This returns a prediction of the expected milk, fat and protein for a cow for the remaining the lactation. Also a prediction of this production on the next testday is included and it eventually gives indication of when the peak production is reached.",
+
+  "allOf": [{
+      "$ref": "../resources/icarAnimalEventCoreResource.json"
+    },
+    {
+      "type": "object",
+
+      "properties": {
+        "averagePredictedProduction": {
+          "$ref": "../types/icarMilkingPredictionType.json"
+        },
+        "daysInMilkAtLactationPeak": {
+          "type": "integer",
+          "description": "The days in milk in a lactation when the peak production is expected to occur."
+        },
+        "lactationPeakProduction": {
+          "$ref": "../types/icarMilkingMilkWeightType.json"
+        },
+        "PredictedProductionNextMR": {
```

```suggestion
        "PredictedProductionNextMR": {
```
        "predictedProductionNextMR": {